packages/fluentui/react-northstar-prototypes/src/prototypes/MenuList/MenuGroup.tsx

```diff
@@ -0,0 +1,16 @@
+import * as React from 'react';
+import { MenuGroupContextProvider } from './menuGroupContext';
+
+let count = 0;
+
+export const MenuGroup = (props: any) => {
+  const headerId = React.useRef<string>(`menu-group-${count++}`);
+
+  return (
+    <MenuGroupContextProvider value={{ headerId: headerId.current }}>
+      <div role="group" aria-labelledBy={headerId.current}>
```

Comment:
![A React warning about invalid attribute](https://user-images.githubusercontent.com/14183168/105691793-f714bb80-5efd-11eb-98f1-e38c6321fbbd.png)

Reply:
We should also handle a scenario when an element with `headerId.current` will not be present